backoff.go

```diff
@@ -27,12 +27,34 @@ import (
 
 // DefaultBackoffConfig uses values specified for backoff in
 // https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md.
+// Deprecated: use ConnectRetryParams instead.
 var DefaultBackoffConfig = BackoffConfig{
 	MaxDelay: 120 * time.Second,
 }
 
 // BackoffConfig defines the parameters for the default gRPC backoff strategy.
+// Deprecated: use ConnectRetryParams instead.
 type BackoffConfig struct {
 	// MaxDelay is the upper bound of backoff delay.
 	MaxDelay time.Duration
 }
+
+// ConnectRetryParams defines the parameters for the backoff during connection
+// retries. Users are encouraged to use this instead of BackoffConfig.
```

Comment:
Can you link to the grpc backoff doc from here please?

Reply:
Done.